test/performance/tools/common.sh

```diff
@@ -132,6 +132,17 @@ function update_cluster() {
   kubectl patch hpa -n knative-serving activator \
         --patch '{"spec": {"minReplicas": 10}}'
 
+  echo ">> Setting up 'prod' config-mako"
+  cat | kubectl apply -f - <<EOF
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: config-mako
+data:
+  # This should only be used by our performance automation.
+  environment: prod
```

Comment:
Can the `prod` literal come from the environment?

Reply:
And likewise any other references. I just don't want folks kicking off these scripts locally and it's polluting our CI dataset

Reply:
This is how it comes from the environment.  What did you have in mind?

Reply:
I figured the value could come a via Prow config.

My general aspiration is to have the scripts generic/configurable enough to be reused across different K8s environments (ie. different GCP projects/AKS/EKS, IaaS's etc.). 

Hence my concern with other CI/CD instances polluting mako. But I realize the GCP project names are hard-coded - so there's more effort required.

Reply:
@dprotaso I expect the distinct ACLs to protect against pollution.  These scripts are far from what I expect their final form will be, but I suspect that in the relatively short term we could make this a flag, if needed.